<ui version="4.0" >
 <class>InvWidget</class>
 <widget class="QWidget" name="InvWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>20</y>
     <width>255</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title" >
    <string>Folder</string>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label_4" >
        <property name="minimumSize" >
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="minimumSize" >
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="authorEdit" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="minimumSize" >
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Header</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="headerEdit" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="label_6" >
          <property name="minimumSize" >
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Comments</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="commentsEdit" />
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QComboBox" name="comboBox" />
      </item>
      <item>
       <widget class="QComboBox" name="colouredCBox" >
        <property name="currentIndex" >
         <number>-1</number>
        </property>
        <property name="insertPolicy" >
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label_3" >
        <property name="minimumSize" >
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit" />
      </item>
      <item>
       <widget class="QCheckBox" name="forUsBox" >
        <property name="text" >
         <string>Only for us</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>16</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="saveButton" >
        <property name="text" >
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
